src/test/java/uk/gov/hmcts/reform/civil/service/NotificationServiceTest.java

```diff
@@ -0,0 +1,66 @@
+package uk.gov.hmcts.reform.civil.service;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.jackson.JacksonAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import uk.gov.hmcts.reform.civil.helpers.CaseDetailsConverter;
+import uk.gov.service.notify.NotificationClient;
+import uk.gov.service.notify.NotificationClientException;
+
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@SpringBootTest(classes = {
+    NotificationService.class,
+    JacksonAutoConfiguration.class,
+    CaseDetailsConverter.class
+})
+public class NotificationServiceTest {
+
+    @Autowired
+    private NotificationService service;
+
+    @MockBean
+    private NotificationClient notificationClient;
```

Comment:
Fun fact (actually: not so fun... took me hours to figure it out on another PR): this is not going to be mocked as it's a class and not an interface. This can cause quite some trouble. Luckily for you, you don't need `@MockBean` in this case if you turn this into a regular JUnit5/mockito test.

I'll show you how.